front/app/components/Form/Components/Layouts/PageControlButtons.tsx

```diff
@@ -63,10 +63,13 @@ const PageControlButtons = ({
             padding={isSmallerThanPhone ? '16px' : '16px 24px'}
           >
             <LanguageSelector
-              top="-200px"
+              className={'open-upwards'}
               mobileRight="auto"
               mobileLeft="auto"
               right="auto"
+              afterSelection={() => {
+                window.location.reload();
+              }}
```

Comment:
This was a weird one that I had trouble fixing. When we change the locale and the website re-loads, it doesn't reload the whole site and the survey modal has some weird behaviour which causes a partially blank screen for the user:

![Screenshot from 2024-04-16 09-15-30](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/23768379-655c-4e76-941c-92cd874848c7)

I figured the safest bet was to just force a reload of the survey page if the locale is changed (which should happen only once by a user anyways).